client/my-sites/importer/author-mapping-item.jsx

```diff
@@ -64,12 +60,16 @@ export default class extends React.PureComponent {
 				<Gridicon className="importer__mapping-relation" icon="arrow-right" />
 				{ ! hasSingleAuthor ? (
 					<AuthorSelector siteId={ siteId } onSelect={ onSelect }>
-						<UserItem user={ selectedAuthor } />
+						<User user={ selectedAuthor } />
 					</AuthorSelector>
 				) : (
-					<UserItem user={ this.getCurrentUser() } />
+					<User user={ this.getCurrentUser() } />
```

Comment:
There's one remaining `this.getCurrentUser()` call here that causes a crash. After changing it to `currentUser` prop access, the site importer starts working.

Reply:
oops! nice catch. fixed.